### ISSUE TITLE
There are some questions about the calculation of reconstruction loss

### ISSUE THREAD

**Issue by zhaolongNCU** [junxia97/Mole-BERT#22]
When I look at the refactoring loss piece of code, I find that the code is different from the formula in the literature?
Why is the cross-entropy loss function used here different from the one in the literature？
`atom_loss = criterion(pred_node, batch.x[:, 0])`
May I ask you why there is any discrepancy between this and the literature? I still don't get it.
Thanks!
